.circleci/config.yml

```
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     macos:
-      xcode: "9.4.1"
+      xcode: "10.1"
```

```suggestion
      xcode: "11.0.0"
```
That should be all we need.

---

Tried to push it myself to speed things up, but apparently I don't have permissions here.

---

Sorry didn't see you added this commit before I've pushed :(